main.go

```diff
@@ -33,7 +37,7 @@ func tallyVulnsByEcosystem(vulns []vulnerabilityAlert, vulnCounts map[string]int
 	}
 }
 
-func newSeverityMap() map[string]int {
+func NewSeverityMap() map[string]int {
```

Comment:
It's not _yet_ referenced outside, but as I was thinking about it, I was thinking this would be something potentially useful to code that wraps this main library.